apps/serverless/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "serverless",
+  "version": "1.0.0",
+  "description": "Serverless framework TypeScript",
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development sls offline start --stage offline",
+    "tail-log": "sls logs -f app -t",
+    "deploy-prod": "cross-env NODE_ENV=production sls deploy --stage prod",
+    "remove-prod": "cross-env NODE_ENV=production sls remove --stage prod",
+    "clean": "rimraf dist .webpack .serverless",
+    "lint": "eslint --ext .js,.ts .",
+    "lint:fix": "eslint --fix  --ext .js,.ts .",
+    "build-types": "tsc --noEmit --pretty",
+    "deploy": "npm run deploy-prod"
+  },
+  "dependencies": {
+    "serverless-http": "^3.0.2"
+  },
+  "devDependencies": {
+    "@types/supertest": "^2.0.12",
```

Comment:
There are some unused deps here, left from when the example included jest and dotenv.

Reply:
fixed